requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Versions are pinned to prevent pypi releases arbitrarily breaking
 # tests with new APIs/semantics. We want to update versions deliberately.
 ansible==2.2.2.0
-boto==2.45.0
+boto==2.34.0
```

Comment:
Is this change supposed to be here or an accident? Not sure why it would be associated with this.